db/db.go

```diff
@@ -80,62 +69,45 @@ func Open(dbPath string) (*DB, error) {
 	openMu.Lock()
 	defer openMu.Unlock()
 
-	dbc, err := sql.Open("sqlite3_with_hook", dbPath)
+	os.MkdirAll(filepath.Dir(dbPath), 0755)
```

Comment:
Also, I don't think this is necessary. The directory tree will be created, right?